frontend/fingertips-frontend/components/pages/search/results/index.tsx

```diff
@@ -11,6 +23,14 @@ export function SearchResults({ indicator }: Readonly<{ indicator: string }>) {
       />
       <H1>Search results</H1>
       <Paragraph>{`You searched for indicator "**${indicator}**"`}</Paragraph>
+      {searchResults.length ? (
+        <UnorderedList className={classes['result']}>
```

Comment:
can use prop `listStyleType="none"` here and don't need the css

Reply:
done